modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -278,6 +278,9 @@ struct CV_EXPORTS_W_SIMPLE Animation
     //! Vector of frames, where each Mat represents a single frame.
     CV_PROP_RW std::vector<Mat> frames;
 
+    //! Indicates whether the animation contains a hidden first frame.
```

Comment:
Maybe:
```
//! image that can be used for the format in addition to the animation or if animation is not supported in the reader (like in PNG).
CV_PROP_RW cv::Mat still_image;
```

Reply:
IMO. 
I agree with vrabaud because it's probably not clear for the API users that first frames has a special meaning.
And if a `still_image` attribute is introduced, we need not worry what does duration is better for hidden frame.
I think it is friendly API design.

(A) Hidden frame/still image is included in frames list.
----

| idx | frames | durations |
|:------:|:-------:|:----------:|
| 0 | still image | ? |
| 1 | 1st frame| xx |
| 2 | 2nd frame | xx |
| 3 | 3rd frame | xx |

-> should durations[0] be 0, 0xFFFF or other value ?

(B) Hidden frame/still image is excluded out frames list.
----

| idx | frames | durations |
|:------:|:-------:|:----------:|
| 0 | 1st frame| xx |
| 1 | 2nd frame | xx |
| 2 | 3rd frame | xx |

and  still-image